test/trackler_test.rb

```diff
@@ -7,4 +7,22 @@ def test_todos
     slugs = %w(banana cherry imbe mango)
     assert_equal slugs, Trackler.todos["fake"].map(&:slug)
   end
+
+  def test_problems_does_not_contain_track_specific_problems
+    Trackler.use_fixture_data
+
+    assert_nil Trackler.problems.detect { |p| p.slug == "snowflake-only" }
+  end
+
+  def test_problems_does_not_contain_track_specific_problems
```

Comment:
I think this test name is a copy/paste. It doesn't reflect the implementation of the test.